physionet-django/events/templates/events/event_home.html

```diff
@@ -92,12 +94,21 @@ <h3>{{ event.title }}</h3>
             <strong>Host: {{ event.host.get_full_name }} </strong><br>
             <small>Created: {{ event.added_datetime|date }}.  {% if event.host == user %} Number of participants: {{ event.participants.all|length }} {% endif %} </small><br>
             <small>Start Date: {{ event.start_date }}. End Date: {{ event.end_date }}.</small><br>
```

Comment:
Sorry, not directly related to this PR, but while we are here...please could you add a line break after the `{{ event.start_date }}.`?